### ISSUE TITLE
 Create and Serialize FQN Traceability Model

### ISSUE THREAD

**Issue by bvarady** [ftsrg/gamma#91]
It would be useful to have a traceability model between model elements and their fully qualified names, along with methods to calculate it for each model element.
E.g. the execution trace contains references to elements of a 'flattened' model, and one possible way of back-annotating those to the original elements is by using FQNs. Also, some model elements currently do not have official FQNs.

**Comment by grbeni** [comment]
See this class for an initial proposal for generating FQNs for elements: https://github.com/ftsrg/gamma/blob/dev/plugins/core/hu.bme.mit.gamma.transformation.util/src/hu/bme/mit/gamma/transformation/util/Namings.xtend

**Comment by bvarady** [comment]
Yes, we are currently using that, but our solution is far from perfect, so we are hoping for an official solution in the future.

**Comment by grbeni** [comment]
Please check out if the solution to #92 solves this issue of yours, too.
